gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -88,7 +88,8 @@
                 java.util.stream.Collectors.*,
                 com.palantir.logsafe.Preconditions.*,
                 com.google.common.base.Preconditions.*,
-                org.apache.commons.lang3.Validate.*"/>
+                org.apache.commons.lang3.Validate.*",
+                "org.apache.spark.sql.functions.*"/>
```

Comment:
This is not correctly formatted.

All the other exclusions are core utilities we use across nearly every product, spark doesn't fit that pattern, and I'm not sure we should open the door for more static imports.

Reply:
Oh good spot - the quotes are wrong!

Reply:
oops, fixed. serves me right for copy pasting.